src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/runtime/Thread.java

```diff
@@ -76,23 +72,6 @@ public int suspendFlags() {
     return (int) suspendFlagsField.getValue(addr);
   }
 
-  public boolean isExternalSuspend() {
```

Comment:
I assume a new SA API for thread suspension is needed here.

Reply:
If it is important to ask if a thread is suspended.
The native stack-traces would easily show this.